native/src/components/AnchorIcon.tsx

```diff
@@ -5,22 +5,27 @@ import styled from 'styled-components/native'
 
 import { isRTL } from '../constants/contentDirection'
 
-const Icon = styled(MaterialIcon)`
+const Icon = styled(MaterialIcon)<{ disabled: boolean }>`
   font-size: 30px;
+  color: ${props => (props.disabled ? props.theme.colors.textDisabledColor : props.theme.colors.textColor)};
 `
 
 type PropsType = {
   name: string
   scrollViewRef: React.ElementRef<typeof ScrollView> | null
   isLeftAnchor: boolean
+  updatePosition: (position: 'start' | 'end') => void
+  disabled: boolean
 }
 
-const AnchorIcon = ({ name, scrollViewRef, isLeftAnchor }: PropsType): ReactElement => {
+const AnchorIcon = ({ name, scrollViewRef, isLeftAnchor, updatePosition, disabled }: PropsType): ReactElement => {
   const onAnchorPress = (): void => {
     if (isLeftAnchor) {
       scrollViewRef?.scrollTo({ x: 0, y: 0, animated: true })
+      updatePosition('start')
```

Comment:
as far as I can tell these calls are not needed, `scrollTo` triggers the `handleScroll` method already, therefore it should be possible to remove this function completely
```suggestion
```

Reply:
removed.